engine/classes/Elgg/EmailService.php

```diff
@@ -127,6 +122,19 @@ public function transport(Email $email) {
 			// which influences the output of the header in the message
 			$message->getHeaders()->addHeaderLine("{$name}: {$value}");
 		}
+		
+		// set Subject
+		$subject = elgg_strip_tags($email->getSubject());
+		$subject = html_entity_decode($subject, ENT_QUOTES, 'UTF-8');
```

Comment:
Please add tests for these changes

Reply:
this is not a change, this has always been the case

Reply:
ps: this is tested https://github.com/Elgg/Elgg/blob/master/engine/tests/phpunit/unit/Elgg/EmailServiceUnitTest.php#L28